scheduler/conf.c

```diff
@@ -3381,6 +3382,26 @@ read_cupsd_conf(cups_file_t *fp)	/* I - File to read from */
         cupsdLogMessage(CUPSD_LOG_WARN, "Unknown LogLevel %s on line %d of %s.",
 	                value, linenum, ConfigurationFile);
     }
+    else if (!_cups_strcasecmp(line, "stripuserdomain") && value)
```

Comment:
Don't add manual code for boolean options. Just add the StripUserDomain directive to the `cupsfiles_vars` array near the beginning of this source file with the `CUPSD_VARTYPE_BOOLEAN` annotation, e.g.:

```c
  { "StripUserDomain",	&StripUserDomain,	CUPSD_VARTYPE_BOOLEAN },
```